tables/endpointsecurity/src/processeventstableplugin.cpp

```diff
@@ -55,11 +55,11 @@ ProcessEventsTablePlugin::schema() const {
     { "user_id", IVirtualTable::ColumnType::Integer },
     { "group_id", IVirtualTable::ColumnType::Integer },
     { "platform_binary", IVirtualTable::ColumnType::Integer },
-    { "signing_id", IVirtualTable::ColumnType::Integer },
-    { "team_id", IVirtualTable::ColumnType::Integer },
-    { "cdhash", IVirtualTable::ColumnType::Integer },
-    { "path", IVirtualTable::ColumnType::Integer },
-    { "cmdline", IVirtualTable::ColumnType::Integer }
```

Comment:
Was at first confused by how this was related to the topic of this change, but see how this was wrong before. Good we have a test catching an incorrect schema now.

While you are touching this you could enable clang-format for the block to give it its missing space at the beginning of the line.